youtube-client/src/app/auth/pages/admin/admin.component.html

```diff
@@ -0,0 +1,70 @@
+<div class="admin">
+  <div class="admin-wrapper">
+    <h3 class="admin-title">Create new card</h3>
+      <form class="admin-form" [formGroup]="adminForm" (ngSubmit)="addCard()">
+
+        <mat-form-field class="admin-form-full-width" appearance="fill">
+          <mat-label>Title</mat-label>
+          <input type="text" matInput formControlName="title">
+          <mat-error *ngIf="adminForm.get('title')?.hasError('required')">
```

Comment:
You should place this in a helper function as it is repeated multiple times in your template